templates/views/views-view-list.html.twig

```diff
@@ -17,16 +17,16 @@
  */
 #}
 {% if attributes -%}
-  <div{{ attributes }}>
+  <div {{ attributes }}>
 {% endif %}
   {% if title %}
     <h3>{{ title }}</h3>
   {% endif %}
-
+{#Prettier Ignore#}
   <{{ list.type }}{{ list.attributes }}>
 
     {% for row in rows %}
```

Comment:
Nice!

Reply:
I've removed this as it does not have any effect. 
The correct ignore syntax is:   {# prettier-ignore #} or {# prettier-ignore-attribute #} or {# prettier-ignore-attribute #}